cmd/aro/operator.go

```diff
@@ -153,26 +154,26 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 			arocli, mcocli, dh)).SetupWithManager(mgr); err != nil {
 			return fmt.Errorf("unable to create controller DnsmasqMachineConfigPool: %v", err)
 		}
-		if err = (node.NewNodeReconciler(
+		if err = (node.NewReconciler(
 			log.WithField("controller", controllers.NodeControllerName),
-			kubernetescli, arocli)).SetupWithManager(mgr); err != nil {
+			arocli, kubernetescli)).SetupWithManager(mgr); err != nil {
 			return fmt.Errorf("unable to create controller Node: %v", err)
 		}
 		if err = (azurensg.NewReconciler(
 			log.WithField("controller", controllers.AzureNSGControllerName),
-			arocli, maocli, kubernetescli)).SetupWithManager(mgr); err != nil {
+			arocli, kubernetescli, maocli)).SetupWithManager(mgr); err != nil {
 			return fmt.Errorf("unable to create controller AzureNSG: %v", err)
 		}
-		if err = (machine.NewMachineReconciler(
+		if err = (machine.NewReconciler(
 			log.WithField("controller", controllers.MachineControllerName),
-			maocli, arocli, isLocalDevelopmentMode, role)).SetupWithManager(mgr); err != nil {
+			arocli, maocli, isLocalDevelopmentMode, role)).SetupWithManager(mgr); err != nil {
 			return fmt.Errorf("unable to create controller Machine: %v", err)
 		}
 	}
 
 	if err = (checker.NewReconciler(
 		log.WithField("controller", controllers.CheckerControllerName),
-		maocli, arocli, kubernetescli, role)).SetupWithManager(mgr); err != nil {
+		arocli, kubernetescli, maocli, role)).SetupWithManager(mgr); err != nil {
 		return fmt.Errorf("unable to create controller InternetChecker: %v", err)
```

Comment:
I know this isn't part of the change, but the controller "Checker" (pkg/operator/controllers/const.go) isn't named "InternetChecker".  Because you are in here, changing this would help with consistency as this controller also checks the service principal, etc.

Reply:
Good find. I'll add a separate commit with this.